cmd/kured/main.go

```diff
@@ -114,6 +119,16 @@ func main() {
 	}
 }
 
+func flagcheck(cmd *cobra.Command, args []string) {
```

Comment:
generally the name of the functions are following some kind of Camel case, so I would expect this function to be named flagCheck or flagChecker